pkgs/applications/version-management/smartgithg/default.nix

```
@@ -1,23 +1,23 @@
 { stdenv, fetchurl, lib, makeWrapper
 , substituteAll
 , jre
-, gtk2, glib
+, gtk3, glib, gnome3, wrapGAppsHook, hicolor-icon-theme, gdk_pixbuf
 , libXtst
 , which
 }:
 
 stdenv.mkDerivation rec {
   name = "smartgithg-${version}";
```

```suggestion
  pname = "smartgithg";
```

when attributes `pname` and `version` are present they automatically create `name` like this.